azure-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+trigger:
+  branches:
+    include:
+      - master
+      - develop
+
+pr:
+  branches:
+    include:
+      - master
+      - develop
+  drafts: false
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '14.17.x' 
+  displayName: 'Installi Node 14.17.x'
+
+- script: npm ci
+  displayName: 'Install dependencies'
+
+- script: npm run build
+  displayName: 'Build React webApp'
+
+- task: PublishBuildArtifacts@1
+  displayName: 'Save artifact'
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
```

Comment:
this says "execute only for PullRequests"

Reply:
we need somethinkg like `condition: and(succeeded(), eq(variables['Build.SourceBranchName'], 'master'))`, but that includes `develop` branch in addtion to master

Reply:
TIL: ne stands for not equal

Reply:
I guess it says **not equal** `nee` for pull requests.